### PR TITLE
General Tidy up of Cookies Section

### PR DESCRIPTION
**Pull request by ayumihamsaki**
General Tidy up of Cookies Section, need to update the German and French Lang Files.

Update from https://github.com/OFFLINE-GmbH/oc-gdpr-plugin/issues/9

### CODE REVIEW

**Comment by ayumihamsaki** [comment]
Sorry I can't speak German and French properly, so I added english inside for the extra fields in the lang files.

**Comment by damsfx** [comment]
On spot for French !!

**Comment by ayumihamsaki** [comment]
Gonna make some more adjustments to make it easier to understand and try to sort out the French lang.

File: lang/fr/lang.php [hunk 1]
```diff
@@ -169,15 +169,53 @@
             'default_level'          => 'Ce niveau est présélectionné pour l\'utilisateur.',
         ],
         'tabs'   => [
-            'description' => 'Description',
+            'description' => 'La description',
```
Comment (damsfx) [inline lang/fr/lang.php]: In French, the defined article "LA" is not necessary in this context.

Comment (ayumihamsaki) [reply lang/fr/lang.php]: @damsfx Sorry my French is really bad, I'm limited to counting to ten, saying bonjour and singing frere jacques, lol. Please translate all the english fields I added in to the file.

Comment (damsfx) [reply lang/fr/lang.php]: @ayumihamsaki Don't apologize for your French, I'm myself trying to do my best in English! ;)

For update french lang file see : pull #18 .  
I will update the file as the project progresses.

File: lang/fr/lang.php [hunk 2]
```diff
@@ -169,15 +169,53 @@
...
+            'contact_links_dpo_prompt'    => 'Add new link',            
+            'contact_links' => [
+                'name' => 'Anchor Text',
+                'href' => 'Link Target (absolute URL)',
+            ],
+            'cookie_levels' => [
+                'purpose_prompt'     => 'Add Granular Purpose Levels',
+                'purpose_label'      => 'Granular Purpose Levels',
+                'section1_label'     => 'Helpful Tip for Granular Purpose Levels',
+                'section1_comment'   => 'Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Granular Opt-In for several Processes. This means having an off setting, the having a basic setting, then having a more advanced setting and so on up to a full setting. An example of this could be Google Analytics where we have an Off Setting, then a basic Tracking Setting, then a basic Tracking Setting plus Plugins, then finally a full setting with everything. The number of levels depends on what you are adding.',
```
Comment (tobias-kuendig) [inline lang/fr/lang.php]: > Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Granular Opt-In for several Processes.

Maybe it's my english, but this doesn't sound like a complete sentence to me?

Comment (ayumihamsaki) [reply lang/fr/lang.php]: "It's lawyer / patent speak", better grammar would be:

> Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Use a Granular Opt-In process with our manage levels feature.

Comment (ayumihamsaki) [reply lang/fr/lang.php]: p.s. could you change the english lang file to that. If you think that's a better sentence.

Comment (tobias-kuendig) [reply lang/fr/lang.php]: Thank you for clarifying that!

**Pull request merged**